Tests/TuistLoaderTests/Extensions/TuistTestCase+ManifestMappers.swift

```diff
@@ -168,7 +167,11 @@ extension TuistTestCase {
                 file: StaticString = #file,
                 line: UInt = #line)
     {
-        XCTAssertEqual(arguments.environment, manifest.environment, file: file, line: line)
+        typealias EnvironmentVariable = [String: String]
+
+        // - FIXME: Need help for this.
+        let rawEnvironments: [EnvironmentVariable: Bool] = arguments.environmentVariables.reduce(into: [:]) { _, _ in }
+        let rawEnvironmenstManifest = manifest.environmentVariables.reduce(into: [:]) { _, _ in }
```

Comment:
The goal here is to compare the raw values of the manifest types and the model types, this can be achieved in any number of ways. We can do something along the lines of:

```swift
XCTAssertEqual(arguments.environmentVariables.count, manifest.environmentVariables.count)
zip(arguments.environmentVariables, manifest.environmentVariables).forEach { model, manifest in
   XCTAssertEqual(model.isEnabled, manifest.isEnabled)
   XCTAssertEqual(model.key, manifest.key)
   XCTAssertEqual(model.value, manifest.value)
}
```